src/theme/variables.css

```diff
@@ -277,6 +277,9 @@ body {
 
 .filters {
   grid-area: filters;
+  position: sticky;
```

Comment:
I think this should be desktop only right?

Reply:
Filters are already visible desktop-only which is why I have not written this styling on desktop-only media query.

Reply:
The same component will be used in the mobile view so the base class has to work for both devices.